src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/message/CreateUpdateService.kt

```diff
@@ -42,13 +46,16 @@ class CreateUpdateService(
   }
 
   private fun handlePersonCreation(person: Person): PersonEntity {
-    val personEntity: PersonEntity = personService.createPersonEntity(person)
-    return personService.linkRecordToPersonKey(personEntity)
+    var personEntity: PersonEntity = personService.createPersonEntity(person)
+    personEntity = personService.linkRecordToPersonKey(personEntity)
```

Comment:
It would be nice if we could avoid this reassignment for readability

```suggestion
  val  personEntity = personService.linkRecordToPersonKey(personService.createPersonEntity(person))
```

Reply:
Updated 👍